assets/scss/brandings.scss

```diff
@@ -1193,5 +1186,13 @@ body.hale-colours-variable {
 			}
 		}
 	}
+}
 
+.govuk-footer {
```

Comment:
WHy are we moving this outside the .hale-colours-variable class?

Reply:
This is to make the default colour work - the variable colours were working as intended, but the default wasn't being picked up.  This should always have been applied to all sites, not just those with custom colours.  

I noticed this on Adam's new site which is GDS styled, but it also appears on the Dashboard and elsewhere.

Reply:
Sounds good